source/features/infinite-scroll.tsx

```diff
@@ -65,6 +66,8 @@ void features.add({
 	include: [
 		pageDetect.isDashboard
 	],
-	onlyAdditionalListeners: true,
```

Comment:
I think this still applies. It **only** needs onNewsfeedLoad

Reply:
I added it

Reply:
Otherwise, this feature could be rewritten as well. Drop `findButton` everywhere, and add something like this in `init`:

```js
observe('.ajax-pagination-btn', {
  add(button) {
    inView.observe(button);
  }
});
```

Reply:
I am too scared to touch it its waay beyond my understanding.

Reply:
I did it I have no idea if it works since I dont have enough events 😃

Reply:
Tested on isProfileMainTab, it works